packages/web-components/src/accordion/index.ts

```diff
@@ -1,5 +1,5 @@
 export { Accordion } from './accordion.js';
 export { AccordionExpandMode } from './accordion.options.js';
-export { template as accordionTemplate } from './accordion.template.js';
-export { styles as accordionStyles } from './accordion.styles.js';
-export { definition as accordionDefinition } from './accordion.definition.js';
+export { template as AccordionTemplate } from './accordion.template.js';
+export { styles as AccordionStyles } from './accordion.styles.js';
+export { definition as AccordionDefinition } from './accordion.definition.js';
```

Comment:
Thanks for this! Since these technically change the API of the package exports, can you run `yarn change` from the root of the repository to generate a change file for these? The change type should be `prerelease` and that should unblock the build pipeline and allow it to complete.

Reply:
Done  🚀